src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,18 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Bulldozer bulldozer = new Bulldozer(true, false);
+        Truck truck = new Truck(false, true);
+        Excavator excavator = new Excavator(true, true);
+        Machine[] machines = {bulldozer, truck, excavator};
+        printReult(machines);
+    }
 
+    public static void printReult(Machine[] machines) {
```

Comment:
Watch out for typos.

```suggestion
    public static void printResult(Machine[] machines) {
```